src/components/Shared/NavFooter.js

```diff
@@ -28,6 +29,11 @@ const navPages = [
     link: "/all-work-orders",
     icon: faTruck,
   },
+  {
+    title: "Assets",
```

Comment:
So I don't think we need a link to the asset search page in the nav footer bar. But we can leave it on the homepage for now...

Reply:
👍